.pre-commit-config.yaml

```diff
@@ -40,7 +40,7 @@ repos:
       - id: check-merge-conflict
       - id: check-shebang-scripts-are-executable
       - id: check-symlinks
-        exclude: "^(bin|test|libs/alcotest_ext/.*test)$"
+        exclude: "^(bin|test|libs/testo/.*test)$"
```

Comment:
I'm not sure you need that since this code is now in a submodule. Not sure pre-commit go inside a submodule

Reply:
I'll fix it. This is the result of automatic find-and-replace.